### PR TITLE
Configure annotations to be ignored in comparisons during sync

### PR DESCRIPTION
**Pull request by FxKu**
Finalizes work done in #1485.

Closes #1485 
Closes #1482

### CODE REVIEW

File: pkg/cluster/cluster.go [hunk 1]
```diff
@@ -720,6 +722,70 @@ func (c *Cluster) enforceMinResourceLimits(spec *acidv1.PostgresSpec) error {
...
+	if reason != "" {
+		return true, reason
+	}
+	return false, ""
+
+}
+
+// SameService compares the Services
+func (c *Cluster) compareServices(old, new *v1.Service) (bool, string) {
+	//TODO: improve comparison
```
Comment (sdudoladov) [inline pkg/cluster/cluster.go:765]: iirc it is just a left-over from the previous implementation, we can drop it

File: pkg/cluster/cluster.go [hunk 2]
```diff
@@ -720,6 +722,70 @@ func (c *Cluster) enforceMinResourceLimits(spec *acidv1.PostgresSpec) error {
...
+			reason += fmt.Sprintf(" Added '%s' with value '%s'.", key, new[key])
+		} else if v != new[key] {
+			reason += fmt.Sprintf(" '%s' changed from '%s' to '%s'.", key, v, new[key])
+		}
+	}
+
+	if reason != "" {
+		return true, reason
+	}
+	return false, ""
```
Comment (sdudoladov) [inline pkg/cluster/cluster.go:759]: does 
```golang
return  reason == "", reason
```
work ?

**Comment by FxKu** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Pull request merged**